base/shell/explorer/trayprop.cpp

```diff
@@ -250,16 +253,61 @@ class CStartMenuSettingsPage : public CPropertyPageImpl<CStartMenuSettingsPage>
 
     LRESULT OnInitDialog(UINT uMsg, WPARAM wParam, LPARAM lParam, BOOL &bHandled)
     {
+        HKEY hKey;
+        DWORD dSz;
+        BYTE regData[12];
+        DWORD dType;
         // fix me: start menu style (classic/modern) should be read somewhere from the registry.
```

Comment:
The NoSimpleStartMenu says that the modern start menu is disallowed. It does not specify whether it is enabled. That must be stored somewhere else (I don't know where).